AzureADApplicationRegistration/functions/Set-AzureADApplicationReplyUrls.ps1

```diff
@@ -3,22 +3,26 @@ function Set-AzureADApplicationReplyUrls {
     param (
         [String] 
         [Parameter(Mandatory)]
+        [Alias("AzureTenantIdSecondary")]
```

Comment:
What's secondary mean?

Reply:
The secondary is there to make it backward compatible. You can call the function `Set-AzureADApplicationReplyUrls -AzureTenantIdSecondary` and `Set-AzureADApplicationReplyUrls -AzureTenantId` and it will do the same. `AzureTenantIdSecondary` parameter is used in the new pipelines where we are interacting with two AD tenants - one where we register AAD apps the other where Key Vaults are located.